tests/models/abiotic/test_radiation.py

```diff
@@ -76,7 +76,7 @@ def test_simple_atmospheric_transmissivity():
         ),
         pytest.param(
             DataArray([100, 1000], dims="cell_id"),
-            DataArray([-0.5, 1.5], dims="cell_id"),
+            DataArray([0.5, 1.5], dims="cell_id"),
```

Comment:
This was negative on purpose to raise an error when the sunshine fraction is negative.

Reply:
Did a too broad global replace and messed up this fix.